segments/ifstat_sys.sh

```diff
@@ -4,15 +4,21 @@
 run_segment() {
 	sleeptime="0.5"
 	if shell_is_osx; then
-		iface="en0"
+		iface=$(route get default | grep -i interface | awk '{print $2}')
+		if [ -z "$iface" ]; then
+			iface="en0"
+		fi
 		type="⎆" # "☫" for wlan
 		RXB=$(netstat -i -b | grep -m 1 $iface | awk '{print $7}')
 		TXB=$(netstat -i -b | grep -m 1 $iface | awk '{print $10}')
 		sleep "$sleeptime"
 		RXBN=$(netstat -i -b | grep -m 1 $iface | awk '{print $7}')
 		TXBN=$(netstat -i -b | grep -m 1 $iface | awk '{print $10}')
 	else
-		iface=$(/bin/cat /proc/net/dev | /usr/bin/awk '{if($2>0 && NR > 2) print substr($1, 0, index($1, ":") - 1)}' | /bin/sed '/^lo$/d')
+		iface=$(ip route get 1.1.1.1 | grep -o "dev.*" | cut -d ' ' -f 2)
+		if [ -z "$iface" ]; then
```

Comment:
I think we can expect these binaries to be in the $PATH

Reply:
Do you mean we should use `/sbin/ip route get xxxx` intead of `ip route get xxx`?